db/seeds.rb

```diff
@@ -51,6 +51,7 @@
   {
     title: 'Need groceries',
     description: 'food pls',
+    due_date: 2020-11-26,
```

Comment:
Due date expects a `datetime` instance so an easy way to seed this woudl be something aong the lines of `Time.now + 259_200`, where we initialize a new 'Time' instance and add 3 days (259200 seconds) to it.